lib/avro/builder.rb

```diff
@@ -26,6 +26,11 @@ def self.build_schema(str = nil, filename: nil, &block)
     def self.add_load_path(*paths)
       Avro::Builder::DSL.load_paths.merge(paths)
     end
+
+    # Define extra allowable metadata attributes for fields
+    def self.extra_metadata_attributes(*attrs)
+      Avro::Builder::Field.extra_metadata_attributes(attrs)
```

Comment:
It looks like the spec supports metadata attributes on all JSON object in the serialized Avro schema. Is there a generalized version of this change that supports attributes on constructs other than just fields e.g. records?

Reply:
@jturkel Yes, I can make that change no problem. Would you rather sperate config blocks for declaring the attributes for fields/records vs other objects? Or is all together ok?

```ruby
Avro::Builder.extra_metadata_attributes(:sensitivity_level, :deprecated_by)
```